bin/client.js

```diff
@@ -55,6 +75,39 @@ async function main() {
   });
 }
 
+async function discoverEndpointURL() {
+  if (!endpointURL) {
+    // Attempt to discover the current stack's accept URL if missing option
+    const Serverless = require("serverless");
+    const serverless = new Serverless({ interactive: false });
+    await serverless.init();
+    await serverless.variables.populateService();
+    const stackName = serverless.providers.aws.naming.getStackName();
+    const stackInfo = await serverless.providers.aws.request(
+      "CloudFormation",
+      "describeStacks",
+      { StackName: stackName }
+    );
+    stackInfo.Stacks[0].Outputs.forEach(({ OutputKey, OutputValue }) => {
+      if (OutputKey === "ServiceEndpoint") {
+        endpointURL = OutputValue;
+        console.log(`Discovered endpoint URL: ${endpointURL}`);
+      }
+    });
+  }
+  if (!endpointURL) {
+    throw "Could not discover endpoint URL";
+  }
+  return endpointURL;
+}
+
+// NOTE: This is a tiny image of lmorchard's face used if --image not supplied
+// (this is probably overkill)
+const DEFAULT_IMAGE = Buffer.from(
```

Comment:
Was also thinking maybe we want to use not images but known text values here to make the mock endpoint to yield a positive or negative result